### PR TITLE
Api departures rennes webservice

### PR DESCRIPTION
**Pull request by julienbt**
Update estimated departures using the webservice

### CODE REVIEW

File: cmd/forseti/main.go [hunk 1]
```diff
@@ -288,15 +286,13 @@ func FreeFloating(manager *manager.DataManager, config *Config, router *gin.Engi
 }
 
 func Departures(manager *manager.DataManager, config *Config, router *gin.Engine) {
-	if len(config.DeparturesFilesURI.String()) == 0 || config.DeparturesFilesRefresh.Seconds() <= 0 {
-		logrus.Debug("Departures is disabled")
-		return
-	}
```
[the remark this answers is not in the document]
Comment (julienbt) [reply cmd/forseti/main.go:294]: This feature is not implemented yet?
Requires a future PR

File: internal/departures/readme.md [hunk 2]
```diff
@@ -11,11 +11,25 @@ Run Forseti and call `http://forseti:port/departures`
...
+./forseti --departures-type=sytralrt \
+--departures-files-uri file:///forseti/fixtures/extract_edylic.txt --departures-files-refresh=10s 
 ```
-./forseti --departures-uri file:///forseti/fixtures/extract_edylic.txt --departures-refresh=1s
-```
+
+SERVICE RENNES
+``` bash
+./forseti --departures-type=rennes \
+--departures-files-uri file:///forseti/fixtures/extract_edylic.txt --departures-files-refresh=300s \
```
[the remark this answers is not in the document]
Comment (julienbt) [reply internal/departures/readme.md]: Correction applied
see https://github.com/hove-io/forseti/pull/98/commits/2d4eefc33cccb117f78d1909f324f7366c52998b

[the remark this answers is not in the document]
Comment (kadhikari) [reply internal/departures/readme.md]: :+1:

**Review by kadhikari — APPROVED** [review]
All apis works fine. :+1:

**Pull request merged**